config/default.json

```diff
@@ -3,6 +3,7 @@
 	"port": 3030,
 	"mongodb": "mongodb://localhost:27017/schulcloud",
 	"public": "../public/",
+	"whitelist": ["$regex", "$limit", "$search", "$sort", "$exists"],
```

Comment:
For which case is $regex in users services use? For field pattern matching?
Mostly we map, or filter querys before it go to the models, will it work?
Will it disallow all other querys?

Reply:
It's not yet used in the user service but in other services. The whitelist is basically good for filtering out all non-allowed "$"-operators, so we don't have to map filter the queries manually. The whitelist can of course be also defined individually, this is just a global config.

Is that the answer for your question?

Reply:
Ok, also whitelist is disallow all not listed query parameter.
$regex sounds like an danger query for whitelist. It should remove and only added in services which need it.

Reply:
OK I removed it globally and just use it in the user service. Should be fine then.